### ISSUE TITLE
changing learning rate

### ISSUE THREAD

**Issue by dimrecon** [jongchyisu/mvcnn_pytorch#4]
Hi,

Is there any problem in changing the learning rate and training , changed to 5e-3 from default=5e-5 as for my dataset ,it seems accuracy 1 for sometimes in epoch 1 itself , after changing learning rate , till epoch 2 no such problem. I am having a large training time such as 14 hrs for 1 epoch , cud u pls tell me how to reduce the training time and effectively use ur model for my dataset.

**Comment by dimrecon** [comment]
Another thing i wanted to know is ,should i train from the scratch as my input images are views of the pointcloud or is it better to use a pretrained model . i assume my input data is different from the imagenet data , am i correct. Could you kindly clarify it?

**Comment by jongchyisu** [comment]
I don't know what dataset you are using, and the learning rate / other hyperparatmeres should be tuned depending on each dataset.
